client/data/php-versions/use-php-versions.jsx

```diff
@@ -0,0 +1,44 @@
+import { useTranslate } from 'i18n-calypso';
+
+export const usePhpVersions = () => {
+	const recommendedValue = '8.1';
+	const translate = useTranslate();
+
+	const phpVersions = [
+		{
+			label: '7.3',
+			value: '7.3',
+			disabled: true, // EOL 6th December, 2021
+		},
+		{
+			label: translate( '%s (deprecated)', {
+				args: '7.4',
+				comment: 'PHP Version for a version switcher',
+			} ),
+			value: '7.4',
+			disabled: true, // EOL 1st July, 2024
+		},
+		{
+			label: '8.0',
+			value: '8.0',
+			disabled: true, // EOL 26th November, 2023
+		},
+		{
+			label: translate( '%s (recommended)', {
+				args: '8.1',
+				comment: 'PHP Version for a version switcher',
+			} ),
+			value: recommendedValue,
+		},
+		{
+			label: '8.2',
+			value: '8.2',
+		},
+		{
+			label: '8.3',
+			value: '8.3',
+		},
+	];
```

Comment:
Maybe not for this PR, but it would be nice to fetch the allowed PHP versions instead of using static values.
fbhepr%2Skers%2Sjcpbz%2Sngbzvp.ncv%2Svapyhqrf%2Spynff%2Qngbzvp%2Qncv.cuc%3Se%3Qo6nn38q9%2379-og